src/main/groovy/life/qbic/oncostore/controller/VariantController.groovy

```diff
@@ -90,27 +91,28 @@ class VariantController {
             schema = @Schema(implementation = Variant.class, type = "object")))
     @ApiResponse(responseCode = "400", description = "Invalid variant identifier supplied")
     @ApiResponse(responseCode = "404", description = "Variant not found")
-    HttpResponse<List<Variant>> getVariants(@Nullable ListingArguments args, @Nullable String format, @QueryValue(defaultValue = "false") @Nullable
-            Boolean withConsequences) {
+    HttpResponse<List<Variant>> getVariants(@Nullable ListingArguments args, @Nullable String format, @QueryValue
```

Comment:
The method looks good. But it is not used?

Reply:
This is triggered automatically when the corresponding endpoint is called